### ISSUE TITLE
Sound dropouts with log messages Bufferoverruns / Bufferunderruns

### ISSUE THREAD

**Issue by datiscum** [FreeRDP/FreeRDP#7005]
Sound dropouts with log messages Buffer overruns/underruns

I have seen several problems with sound playback here, but I don't know if they are the same problems as mine. 
I use FreeRDP on a Raspberry PI4 with PulseAudio and as server Ubuntu 20.04 with xrdp. I can see videos in Firefox in fullscreen resolution (2560x1440) but the sound has a lot of dropouts.

The main problem is a bufferoverrun which is followed by a bufferunderrun. I have adjusted one line in the sources of version "2.0.0.0-dev5", see below. After that, I had no more problems for several months.
Today I had to make the same changes again in the sources of the latest version because I had sound problems again after the installation. I don't know if it's correct or if it's better to change other parameters, but maybe someone who knows more than me can take a look at it. 

FreeRDP/channels/rdpsnd/client$ joe rdpsnd_main.c
static BOOL rdpsnd_detect_overrun(rdpsndPlugin* rdpsnd, const AUDIO_FORMAT* format, size_t size)
{

   /* Maximum allow duration calculation */
//maxDuration = duration * 2 + rdpsnd->latency;
  maxDuration = duration * 6 + rdpsnd->latency; // *6 works well

**Comment by akallabeth** [comment]
ok, and you´re using:
1. Which sound backend?
3. Which version of `freerdp`?

And you´re connected by a:
1. Laggy?
2. High Bandwidth?

connection?

**Comment by datiscum** [comment]
Client Raspberry PI4 with PulseAudio and XRdp sink PulseAudio.
Server "pulseaudio-module-xrdp"

First changed version of Freerdp "2.0.0-dev5" with which it worked.
Today I used the current code with Git. 3.0.0-dev is displayed.
I changed it a long time ago and then forgot about it. Only when I compiled the new version today did I notice the sound problem again.

The server and client are in the local network with a gigabit connection.
